books/free-programming-books-langs.md

```diff
@@ -1962,6 +1962,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 #### Pandas
 
 * [Learn Pandas](https://bitbucket.org/hrojas/learn-pandas) - Hernan Rojas (0.18.1)
+* [Listen Data](https://www.listendata.com/2017/12/python-pandas-tutorial.html) - Ekta Aggarwal (HTML)
```

Comment:
- Using title as it is
- Alphabetize to solve linter warnings

```suggestion
* [Best Pandas Tutorial \| Learn with 50 Examples](https://www.listendata.com/2017/12/python-pandas-tutorial.html) - Ekta Aggarwal (HTML)
* [Learn Pandas](https://bitbucket.org/hrojas/learn-pandas) - Hernan Rojas (0.18.1)
```

Reply:
Thank you for letting me know!
I've committed the suggested edits.